src/migrations/20171005123242_orders.ts

```diff
@@ -3,7 +3,7 @@ import * as Knex from "knex";
 exports.up = async (knex: Knex): Promise<any> => {
   return knex.schema.dropTableIfExists("orders").then(async (): Promise<any> => {
     return knex.schema.createTable("orders", (table: Knex.CreateTableBuilder): void => {
-      table.string("orderId", 42).primary().notNullable();
```

Comment:
Huh, so sqlite let us get away with storing too much in that column

Reply:
Yeah, SQLite doesn't really pay attention to the types you give it. It will store whatever you shove at it in the column, its kinda weird.